clients/src/main/java/org/apache/kafka/common/resource/ResourcePattern.java

```diff
@@ -89,7 +89,7 @@ public ResourcePatternFilter toFilter() {
 
     @Override
     public String toString() {
-        return "ResourcePattern(resourceType=" + resourceType + ", name=" + ((name == null) ? "<any>" : name) + ", patternType=" + patternType + ")";
+        return "ResourcePattern(resourceType=" + resourceType + ", name=" + name + ", patternType=" + patternType + ")";
```

Comment:
Good catch. It's nice to not always have to check for null.

I really wish we had removed null from ResourceFilterPatterns as well (but it's too late for that now, I guess...)